OrbitGl/ThreadTrack.cpp

```diff
@@ -206,3 +219,19 @@ std::string ThreadTrack::GetTooltip() const {
   return "Shows collected samples and timings from dynamically instrumented "
          "functions";
 }
+
+float ThreadTrack::GetHeight() const {
+  TimeGraphLayout& layout = time_graph_->GetLayout();
+  bool is_collapsed = collapse_toggle_->IsCollapsed();
```

Comment:
I like intermediary results like `is_collapsed`, `collapsed_depth`, `depth` to be const if possible. Not required by style guide though.

Reply:
Of course! Thank you for your suggestion!